sktime/regression/deep_learning/cntc.py

```diff
@@ -250,6 +250,12 @@ def _predict(self, X, **kwargs):
         preds = self.model_.predict([X2, X, X], self.batch_size, **kwargs)
         preds = np.squeeze(preds, axis=-1)
         return preds
+    def _more_tags(self):
```

Comment:
this is how it would work in `sklearn`, but not in `sktime`! here we use the `_tags` attribute (scroll up)